app/views/downgrade/new.html.erb

```diff
@@ -0,0 +1,3 @@
+<h4>Would you like to downgrade your account back to standard?</h4>
+<p>Remember, you will have to pay another $15 to get premium features back on your account, and your private wikis will become public.</p>
+<%= link_to "Downgrade Your Account", downgrade_index_path, method: :create, class: 'btn btn-danger', data: {confirm: 'Are you sure you want to downgrade?'} %>
```

Comment:
instead of `method: :create` it should be `method: :post` - that will ensure you actually send a POST request - as it stands, you're making a GET request